styles.css

```diff
@@ -1,3 +1,150 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap");
+
 body {
-  background-color: white;
+  margin: 0;
+  min-height: 100vh;
```

Comment:
better to apply this to a container rather than the body itself, that way you have more flexibility about removing the container later on if you don't want it.